src/Diagram.js

```diff
@@ -383,7 +385,20 @@ class Diagram extends React.PureComponent {
     return <Minimap {...minimapProps} />;
```

Comment:
Can remove sending zoom to minimap props. Instead handle everthing related to zoom inside flowchart lib.